ELEC Virtual Lab/Assets/Scripts/CheckLogin.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using TMPro;
+using UnityEngine.SceneManagement;
+
+public class CheckLogin : MonoBehaviour
+{
+    [SerializeField]
+    private TMP_InputField Name;
+
+    [SerializeField]
+    private TMP_InputField StudentID;
+
+    [SerializeField]
+    private TMP_InputField ClassCode;
+
+    [SerializeField]
+    private TMP_InputField LabSection;
+
+    public void checkLoginInfo()
```

Comment:
The check is good, but it would be tedious to have to write every student, so if possible make a class called Student kinda like this

	public string _name;
	public int _studentID;
	public string _classCode;
	public string _labSection;


	Student(string name, int studentID, string classCode, string labSection)
	{
		_name = name;
		_studentID = studentID;
		_classCode = classCode;
		_labSection = labSection;
	}
	
	
	Then back in the CheckLogin class
	
	you can do List<Student> students = new List<students>();
	students.AddRange(new List<Student>
	{
		new Student("Zachary",40004528,"A","AI")
		new Student(...)
	}
	
	Then for the check you can do 
	
	bool canEnterLab = false;
	
	for(int i = 0 ; i < students.count ; i++)
	{
		if(name.toString.equals(students[i]._name) 
		&& studentID.toString == students[i]._studentID 
		&& classCode.toString.equals(students[i]._classCode
		&& labSection.toString.equals(students[i]._labSection)
			{
				canEnterLab = true;
			}
			
	}
	
	if(canEnterLab)
		SceneManager.LoadScene(2);